meganotes-master/app/notes/notes-form.js

```diff
@@ -0,0 +1,39 @@
+(function() {
```

Comment:
TODO:Bindable members up top: $scope.clearForm = clearForm, save, delete (missing?),
Singular responsibility (controller in separate file)